### DISCUSSION TITLE
Polestar App and API have both stopped displaying charging status for Polestar 3

### DISCUSSION THREAD

**Discussion by BJReplay** [pypolestar/polestar_api#277] [General]
As of this morning (21 Dec 2024 11:00 GMT+11) both the polestar app (Android) and the API have stopped displaying the charger status, the plugged status, and a number of (but not all) charging attributes.

Given that both the app and the API are displaying these issues, I've started a discussion rather than raising an issue, as either my car (1.1.23) or the API in general is playing up.

Is anyone else noticing issues with charging status showing up correctly in their app or the API?

The only API field that is correct is Charging Status:

![image](https://github.com/user-attachments/assets/322e36e2-4862-433b-a1ea-bfa3d7ad18f2)

The App shows the charge percentage increasing but doesn't show that the charger is connected.

![Screenshot_20241221_130004](https://github.com/user-attachments/assets/540027a0-a5dd-43c3-9a30-0c0a848698e9)

**Comment by BeauGiles** [comment]
> as either my car (1.1.23)

Assuming this is a Polestar.. 3? 4?

(as that doesn't look like a Polestar 2 version)

My Polestar 2 is working fine, just plugged it in to confirm - status is correct and updating.

**Reply by BJReplay** [reply]
> Assuming this is a Polestar.. 3

Sorry, yes, a Polestar 3.

**Comment by jschlyter** [comment]
Does it work now? Both the app and the API works fine for me showing charging status as expected.

**Comment by BJReplay** [comment]
I will try shortly.  In the meantime, I also received this notification of a required repair:

The unit of 'Polestar 4179 Estimated Charging Time Minutes to Target Distance' (sensor.polestar_4179_estimated_charging_time_minutes_to_target_distance) changed to '' which can't be converted to the previously stored unit, 'min'. with a prompt to update the history.  I have, for the moment, ignored it.

A reload of the integration did not make the repair go away.

**Comment by BJReplay** [comment]
Both app and API are working today.

And, on plugging in the charger, the Estimated Charging Time Minutes to Target Distance reverted to minutes.

I'll put it down to a glitch either in the cloud or my vehicle.

Given the issues I've had with the vehicle, more likely the vehicle than the cloud.
